6/root/code/.gitkeep

```diff
@@ -0,0 +1,20 @@
+FROM outrigger/servicebase
```

Comment:
Does the content of this .gitkeep get used?

Reply:
Removed this from the #4